nemo_skills/pipeline/utils/exp.py

```diff
@@ -412,6 +417,10 @@ def add_task(
         sandbox_port = get_free_port(strategy="random")
 
     env_vars = get_env_variables(cluster_config)
+    # If not set by env variable, try to resolve from config
+    if skip_hf_home_check is None:
+        skip_hf_home_check = cluster_config.get("skip_hf_home_check", False)
```

Comment:
Why don't we simply write,
```
skip_hf_home_check = cluster_config.get("skip_hf_home_check", skip_hf_home_check)
```

and keep the default in above arguments as `False`?

Reply:
The function arg should override env arg. So only if function arg is unset (None), then resolve via env var next, and if that also doesnt exist, fall back to False (original default).